packages/core/src/browser/progress-status-bar-item.ts

```diff
@@ -30,6 +30,14 @@ export class ProgressStatusBarItem implements ProgressClient {
     @inject(StatusBar)
     protected readonly statusBar: StatusBar;
 
+    protected messagesByProgress = new Map<string, string | undefined>();
```

Comment:
I understand that you just moved this map from one place to another one and it's not related to your PR.
Just to clarify for me: I don't see when the map is cleaned.
So
is it not critical for `ProgressStatusBarItem`?
 or
nevertheless it makes sense to clean up the map at some steps?

Reply:
@AlexTugarev Should we delete it when progress is done?

Reply:
👍